.meta/_partials/descriptions/_datadog.toml

```diff
@@ -1,5 +1,5 @@
 description = """\
-[Datadog][urls.datadog] is a monitoring service for cloud-scale applications, \
-providing monitoring of servers, databases, tools, and services, through a \
-SaaS-based data analytics platform.\
+[Datadog][urls.datadog] is a managed monitoring service for cloud-based \
+applications. It allows you to monior servers, databases, and other services \
+through a SaaS-based monitoring platform.\
```

Comment:
I'd probably avoid abbreviations and do `Software-as-a-Service` if it isn't previously defined and abbreviated in the content.